v2/types.go

```diff
@@ -117,17 +117,30 @@ type ServiceInstanceSchema struct {
 // ServiceBindingSchema represents a plan's schemas for the parameters
 // accepted for binding creation.
 type ServiceBindingSchema struct {
-	Create *InputParametersSchema `json:"create,omitempty"`
+	Create *RequestResponseSchema `json:"create,omitempty"`
 }
 
 // InputParametersSchema requires a client API version >=2.13.
 //
 // InputParametersSchema represents a schema for input parameters for creation or
 // update of an API resource.
 type InputParametersSchema struct {
+	// The schema definition for the input parameters. Each input parameter
+	// is expressed as a property within a JSON object.
 	Parameters interface{} `json:"parameters,omitempty"`
 }
 
+// RequestResponseSchema requires a client API version >=2.14.
+//
+// RequestResponseSchema contains a schema for input parameters for creation or
+// update of an API resource, and a schema for the credentials returned by the
+// broker
+type RequestResponseSchema struct {
+	*InputParametersSchema
```

Comment:
Seems like we should embed the type rather than a pointer to it - WDYT?

Reply:
Fixed it.